plexapi/media.py

```diff
@@ -704,13 +704,15 @@ class Marker(PlexObject):
     """
     TAG = 'Marker'
 
+    def __repr__(self):
+        name = self._clean(self.firstAttr('type'))
+        start = self._clean(self.firstAttr('start'))
+        end = self._clean(self.firstAttr('end'))
+        return '<%s>' % ':'.join([p for p in [self.__class__.__name__, name, start, end] if p])
```

Comment:
Since we are adding a repr, lets show the start and end in "human" time.

````
def sec_to_hh_mm_ss(seconds):
    return time.strftime("%H:%M:%S", time.gmtime(seconds))
````

Reply:
Seconds: `[<Marker:intro:139770:230481>]`
Human:  `[<Marker:intro:14:49:30:16:01:21>]`

Reply:
Looks like it's actually milliseconds. I've adjusted the code to fit. 

```python
def mill_sec_to_hh_mm_ss(milliseconds):
    if not isinstance(milliseconds, int):
        milliseconds = cast(int, milliseconds)
    hhmmsssm = timedelta(milliseconds=milliseconds)
    return str(hhmmsssm).split(".")[0]
```

Reply:
Cant you change it to this one, make the format more correct and is more precice
````
def ms_to_hh_mm_ms(ms):
    r = datetime.datetime.fromtimestamp(ms/1000)
    f = r.strftime("%H:%M:%S.%f")
    return f[:-2]
````

Reply:
Also Change the repr so its [<Marker:intro: 00:02:19 - 00:03:50>]